common/src/main/java/org/apache/rocketmq/common/UtilAll.java

```diff
@@ -461,33 +461,9 @@ private static boolean ipCheck(byte[] ip) {
         if (ip.length != 4) {
```

Comment:
Is it a necessary judgment here? Although you use the InetAddressValidator, I would like to recommend you solve the problem thoroughly. BTW, Hopefully, you could continue to optimize this class. There are many implementations that can be replaced by existing libs. I would like to follow your process.

Reply:
As a utility class method, I think this judgment code is necessary to exclude illegal IPV4 addresses in the first place. Regarding other method optimizations for this class, I will submit other PR in the future.